adding_species_manuscript.tex

```diff
@@ -0,0 +1,847 @@
+demographic model with a new estimate of mutation rate may lead to
+unrealistic levels of genetic diversity. In some situations the
+difference may come down to only a time scaling, so that in principle a
+fitted demographic model may be translated to an equivalent one that
+uses a different generation time or mutation rate. However, that process
+would be opaque and fraught with potential for error.
+
+When possible, all rate, generation time, and demographic models should
+be drawn from the same study. Mixing sources is not ideal, but sometimes
+necessary and should be approached with deliberation. In this situation
+it is especially important to discuss these choices and their potential
+consequences with other researchers. (TODO: more?)
+
+\hypertarget{what-if-we-dont-know-everything}{%
+\paragraph{What if we don't know
+everything?}\label{what-if-we-dont-know-everything}}
+
+In our experience, a commonly encountered missing ingredient for a
+population genetic simulations (neutral and non-neutral) in non-model
+species is a chromosome-level assembly. Many species' genome assemblies
+are composed of many relatively small contigs whose relation to each
+other is unknown. There are two main ways to deal with this.
+
+One option would be to effectively treat these contigs as chromosomes,
+simulating them either together (unlinked) or separately. This feels
+``realistic'', as the simulated chromosomes exactly match (in length, at
+least) the real data. The alternative is to leave aside such strict
+matching, instead simulating an anonymous chromosome from which patterns
+of genetic variation can be extracted (if important, in chunks of size
+similar to the contigs). The latter is in fact usually more realistic,
+since this includes linkage between sites that share a chromosome but
+may be on different contigs, that can affect patterns of neutral genetic
+diversity and perhaps more crucially, linked selection. Contig-level
+assemblies are sometimes annotated, and simulations of regions around
+specific genes or genomic features (eg. exons) may be of interest.
+However using the precise location of genes on many, unlinked contigs is
+a false precision because of the importance of modeling linked selection
+across contigs. For this reason, the question of ``realism'' is most
+relevant for those species having chromosome-level assemblies. In our
+experience it is uncommon that a species have widely-used demographic
+models but not a chromosome-level assembly.
+
+Nonetheless, some of these ingredients will likely be missing for many
+non-model species. Table 1 displays some considerations and solutions
+For example, if all the parameters above are available, except for a
+chromosome level assembly, then one might choose to include only the
+larger contigs.
+
+Table 1. Guide to missing parameters.
+
+% Please add the following required packages to your document preamble:
+% \usepackage{booktabs}
+\begin{table}[]
```

Comment:
table needs linebreaks, but should these be added later when the content is finalized?

Reply:
we can deal with these later when we have the content there